src/modules/media/components/DynamicActionMenu/DynamicActionMenu.module.scss

```diff
@@ -0,0 +1,14 @@
+@use "styles/abstracts" as *;
+
+.c-dynamic-action-menu {
+	@extend %reset-list;
+
+	width: 100%;
+	min-width: 0; // prevent overflow in flex container
+	display: flex;
+	column-gap: $spacer-xs;
```

Comment:
Zal op termijn ook gebruikt moeten worden in de navigatie, best om dus spacing weg te laten en op child-niveau te plaatsen